### PR TITLE
[housekeeping] - Fix minor syntax issues

### PR DESCRIPTION
**Pull request by kidster**
**Quality audit reminder**

Doc team: While logged in using your O365 account, complete the quality checklist at http://bit.ly/2hwvBKX. Use the checklist for significant doc changes or additions submitted by you or a non-doc staff contributor.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-5874fd0451/docs/cloud-identity/v2/)

File: api-docs/api-reference/methods/users/delete-user-api-key-credentials-v2.0-users-userid-os-ksadm-credentials-rax-kskey.rst [hunk 1]
```diff
@@ -9,15 +9,14 @@ Delete user API key credentials
...
-An API key is a credential; you can delete a user's API key with this call.
-To delete a credential from a user, specify the user ID in the request.
-If you know the user name but not the ID, use
 the
-:ref:`list users <get-list-users-v2.0>` operation to
 obtain complete
+An API key is a credential; you can delete a user's API key with this call. To
+delete a credential from a user, specify the user ID in the request. If you
+know the user name but not the ID, use
the
```
Comment (rackerbob) [inline api-docs/api-reference/methods/users/delete-user-api-key-credentials-v2.0-users-userid-os-ksadm-credentials-rax-kskey.rst]: need space 'usethe'

File: api-docs/api-reference/methods/users/delete-user-api-key-credentials-v2.0-users-userid-os-ksadm-credentials-rax-kskey.rst [hunk 2]
```diff
@@ -9,15 +9,14 @@ Delete user API key credentials
...
+:ref:`list users <get-list-users-v2.0>` operation to
obtain complete
```
Comment (rackerbob) [inline api-docs/api-reference/methods/users/delete-user-api-key-credentials-v2.0-users-userid-os-ksadm-credentials-rax-kskey.rst]: need space - 'toobtain'

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-a8b2211af6/docs/cloud-identity/v2/).

**Pull request merged**